### ISSUE TITLE
Works with Plots.jl, but warns: "InspectDR not known to be compatible with current version of Plots.jl"

### ISSUE THREAD

**Issue by ma-laforge** [ma-laforge/InspectDR.jl#13]
### Description

InspectDR includes some code to ensure it can work with the installed version of Plots.jl.  If InspectDR detects that it is more recent than the last time it was known to be compatible with Plots.jl, it displays the following warning:

```
InspectDR not known to be compatible with current version of Plots.jl
[...]
```

This warning occurs in case the API of InspectDR changes in a way that Plots.jl can no longer control it properly.  The message also provides a combination of Plots.jl/InspectDR.jl that is known to be compatible (known to work).  The intent is so that the user does not get discouraged trying to find a solution that works.

### Problem

This compatibility solution is not very good and requires constant updating.  Failure to do so generates a warning message about compatibility that is not actually relevant.

A better solution to this problem ideally should be found.
